Scripts/NodePort.cs

```diff
@@ -19,9 +19,22 @@ public enum IO { Input, Output }
             }
         }
 
-        public IO direction { get { return _direction; } }
-        public Node.ConnectionType connectionType { get { return _connectionType; } }
-        public Node.TypeConstraint typeConstraint { get { return _typeConstraint; } }
+        // public IO direction { get { return _direction; } }
+        // public Node.ConnectionType connectionType { get { return _connectionType; } }
+        // public Node.TypeConstraint typeConstraint { get { return _typeConstraint; } }

```

Comment:
This is git. We don't want any commented code.

Reply:
I know, and this is a blatant oversight on my part. How embarrassing... 😅

Reply:
Now removed.